nearcache/nearcache_it_test.go

```diff
@@ -316,6 +317,20 @@ func TestMapRemove_WithNearCache(t *testing.T) {
 	})
 }
 
+func TestMapRemoveAll_WithNearCache(t *testing.T) {
+	tcx := newNearCacheMapTestContext(t, nearcache.InMemoryFormatBinary, true)
+	tcx.Tester(func(tcx it.MapTestContext) {
+		t := tcx.T
+		m := tcx.M
+		const size = int32(100)
+		populateMap(tcx, size)
+		populateNearCache(tcx, size)
+		err := m.RemoveAll(context.Background(), predicate.Less("__key", int32(50)))
+		require.NoError(t, err)
+		require.Equal(t, int64(50), m.LocalMapStats().NearCacheStats.OwnedEntryCount)
```

Comment:
Owned entry count should be 0 after the near cache is deleted. I think near cache was not setup for this map.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-go-client/pull/873/commits/59d4abf8629e69964b3905af346d81a7dbfb6fd3.